src/OpenVRTrackersDriver.cpp

```diff
@@ -302,17 +302,18 @@ openvr::DevicesManager::pose(const std::string& serialNumber) const
         return std::nullopt;
     }
 
+    std::vector<vr::TrackedDevicePose_t> poses;
```

Comment:
I tried to add it to the class `openvr::DevicesManager` defined in `OpenVRTrackersDriver.h` but it I had to include `openvr.h` in that file instead of `OpenVRTrackersDriver.h`. For some reason, it's not able to recognize the file. I don't know what I'm missing.

Reply:
I think you can add it to the ``DevicesManager::Impl`` class defined in the ``.cpp``.